SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.cxx

```diff
@@ -255,6 +255,17 @@ const TArray<Worker_ComponentId> KnownEntityAuthorityComponents = { POSITION_COM
                                                                     DEPLOYMENT_MAP_COMPONENT_ID, STARTUP_ACTOR_MANAGER_COMPONENT_ID,
                                                                     GSM_SHUTDOWN_COMPONENT_ID,	 VIRTUAL_WORKER_TRANSLATION_COMPONENT_ID};
 
```

Comment:
Completely unrelated to the change you're making here: Why does this file have the extension `.cxx` and not `.cpp` like all the others? My Rider in particular doesn't believe it is part of the project :O

Reply:
Previously SpatialConstant.h contained all the "heavy" types. I moved them to another file, to prevent them to be instantiated into every single translation unit. But since they were inlined, potentially every single GDK modules needs them. DllExporting them is a nono IMO, so they could fit into a .lib. But I had no idea how to fit that into the build system.
Next best thing, a cxx file, ignored by the build tool, that you explicitly include in a single file for each module.